### PR TITLE
Implement error handling by putting the info into   dev null  silently ignoring it: #88

### LINKED ISSUE #88: Implement error handling by putting the info into __Dev Null_ (silently ignoring it)
## Background
One of the provided `ErrorHandling` implementations. The `ErrorMessage` entries should be ignored

## Feature
Don't write errors anywhere, aggregation is not needed the same `DataFrame` as returned.

### PR DESCRIPTION
**Pull request by TebaleloS**
Closes #88

### CODE REVIEW

File: spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/types/ColumnOrValue.scala [hunk 1]
```diff
@@ -24,7 +24,7 @@ import za.co.absa.spark.commons.sql.functions.null_col
 import scala.language.higherKinds
 
 /**
- * Class to unify a representation of a [[za.co.absa.spark.commons.errorhandling.ErrorMessageSubmit ErrorMessageSubmit]] segments.
+ * +Class to unify a representation of a [[za.co.absa.spark.commons.errorhandling.ErrorMessageSubmit ErrorMessageSubmit]] segments.
```
Comment (benedeki) [inline spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/types/ColumnOrValue.scala:27]: Why the change?

File: spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/implementations/ErrorHandlingIgnoringErrors.scala [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+   * First of the few methods that needs to be coded in the trait implementation
+   * The purpose of this method is to convert the error specification into a [[org.apache.spark.sql.Column spark.Column]] expression
+   *
+   * @param errorMessageSubmit - the error specification
+   * @return - the error specification transformed into a column expression
+   * @group Error Handling
+   * @since 0.6.0
+   */
+  override protected def transformErrorSubmitToColumn(errorMessageSubmit: ErrorMessageSubmit): Column = {
+    lit(null)
```
Comment (benedeki) [inline spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/implementations/ErrorHandlingIgnoringErrors.scala:20]: This works, but let's use `null_col` from `za.co.absa.spark.commons.sql.functions` as suggested. 😉 
It avoids using the `null` value in the code, which some linters might mark as an issue.

**Review by benedeki — COMMENTED** [review]
Also license check is failing.

File: spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/implementations/ErrorHandlingIgnoringErrors.scala (hunk 2 above)
Comment (TebaleloS) [reply spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/implementations/ErrorHandlingIgnoringErrors.scala:20]: I'm on it.

**Review by benedeki — APPROVED** [review]
Nice 😉
